### ISSUE TITLE
Azure Netapp Files - az netappfiles list-mount-targets

### ISSUE THREAD

**Issue by rebecca-makar** [Azure/azure-cli#12987]
Running this command through automation on azure devops pipelines on ubuntu machine:
```
az netappfiles list-mount-targets --resource-group ${RESOURCE_GROUP_NAME} \
        --account-name ${ANF_ACCOUNT} \
        --pool-name ${ANF_POOL} \
        --volume-name ${ANF_VOL}
```

I get this error: 
```
Command group 'netappfiles' is in preview. It may be changed/removed in a future release.
Azure Error: InvalidResourceType
Message: The resource type 'netAppAccounts/capacityPools/volumes/mountTargets' could not be found in the namespace 'Microsoft.NetApp' for api version '2019-10-01'. The supported api-versions are '2017-08-15,2019-05-01,2019-06-01,2019-07-01,2019-08-01'.
```

It seems around April 9 or 10 that this error started happening. It is not happening on my local machine so I am assuming that the Azure-Cli or Azure-Cli-Extension had a version change that either  recently and that it is installed on the machines that the versions are running. None of my requirements or versions changed. When I was going through cli / extensions changes it looks like there was a release on april 9? I just am not getting a clear error message and nothing in process changed through automation. Since az-cli is already installed on ubuntu machines for azure devops I am assuming something with newest version of that or extension.

Also added apt list to pipelines to get cli version:
 azure-cli/xenial,now 2.3.1-1~xenial all [installed]

**Comment by yonzhan** [comment]
netappfiles

**Comment by yellowhat** [comment]
I am having the same issue, any update on it?

Thanks

**Comment by xpillons** [comment]
Having the same again today with azcli 2.3.1 on a devops agent I've built with the latest azcli, while it works on Cloud shell which is azcli 2.2.0

**Comment by xpillons** [comment]
What is the status on this issue ? it still happening on 2.5.1.

**Comment by rebecca-makar** [comment]
Reverting to azcli 2.2.0 and it works. Any of the newer cli versions it is broke.

**Comment by xpillons** [comment]
@rebecca-makar this workaround is not always possible. CloudShell is on 2.4.0 and can't be reverted I guess

**Comment by rebecca-makar** [comment]
@xpillons  completely agree. Wanted to say where I noticed the difference

**Comment by rebecca-makar** [comment]
@yonzhan any updates on this? Still broken

**Comment by yonzhan** [comment]
@rebecca-makar netappfiles service team should look into this.

**Comment by rebecca-makar** [comment]
@yonzhan how do I contact them?

**Comment by yonzhan** [comment]
@leonardbf could you please help with this?

**Comment by audunn** [comment]
Quick test using Azure CLI docker images shows this working the same in latest and 2.2.0. 

However note that list-mount-targets is due for deprecation/removal in upcoming version of the CLI, as mount target information can be retrieved using volume show. 

az netappfiles volume show -g TestRG -a test-account -p test-pool -v tests-vol-1

`
.
.
  "mountTargets": [
    {
      "endIp": "",
      "fileSystemId": "3fcf5730-6f36-0900-7228-5effdea724bd",
      "gateway": "",
      "ipAddress": "10.14.0.5",
      "mountTargetId": "3fcf5730-6f36-0900-7228-5effdea724bd",
      "netmask": "",
      "provisioningState": "",
      "startIp": "",
      "subnet": ""
    }
  ],
.
.
`

**Comment by ranieuwe** [comment]
Related to #13011 - this seems resolved in version 2.10.1. This can probably be closed.
